lib/repl.js

```diff
@@ -545,6 +545,9 @@ REPLServer.prototype.createContext = function() {
     });
   });
 
+  // Refer: https://github.com/nodejs/node/pull/3729#issuecomment-155460861
```

Comment:
Instead of referring to an issue, how about just explaining why we're doing this.